packages/checkout-storefront/.eslintrc.js

```diff
@@ -1,10 +1,7 @@
 module.exports = {
-  parser: "@typescript-eslint/parser",
+  root: true,
+  extends: ["checkout"],
```

Comment:
Should each package that's not using Next.js extend this config? From what I see `eslint-config-checkout` uses now `next` configuration 🤔

Reply:
Yeah, that's not a problem.